atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -67,11 +69,33 @@ public static synchronized void hydrate(Map<TableReference, byte[]> tableRefToMe
         logArbitrator = SafeLoggableDataUtils.fromTableMetadata(tableRefToMetadata);
     }
 
+    public static synchronized void setAllSafeForLogging(boolean isNewKeyValueServiceAllSafeForLogging) {
```

Comment:
nit: can we name this `accumulate...` or something along those lines? (since setting to `true` won't always lead to behaviour we'd expect with `true` if there was a `false` passed earlier)